### DISCUSSION TITLE
Is it possible to show the section heading when cross referencing the section, instead of e.g. "Section 2"

### DISCUSSION THREAD

**Discussion by Santonia27** [quarto-dev/quarto-cli#10138] [Q&A] [answered]
### Description

![image](https://github.com/quarto-dev/quarto-cli/assets/145111392/b7f54ed2-de66-4a61-aae6-63963e09bc00)

Instead of xyz (Section 2) I would like to cross reference be the word itself, same principle as the cross referencing to a different page. 
I tried the same principle but it didnt work:
[quickbuild] (@sec-quickbuild)  (I put the space after [] for demonstration purposes, so the cross-referencing does not happen)

**Comment by mcanouil** [comment, marked as answer]
Not fully sure I understand.
Section are anchored (html) by default.

<table>
<tr><th>Input</th><th>Output</th></tr>
<tr><td>

````qmd
---
title: "Quarto Playground"
format: html
---

## First Section {#sec-first}

This is a playground for Quarto.

{{< lipsum 1 >}}

## Second Section

See [text @sec-first]

or

See [Text](#sec-first)
````

</td><td>

<img width="714" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/f3660da4-2361-4a2f-9d05-de80750fe54b">

</td></tr>
</table>

**Reply by Gewerd-Strauss** [reply]
In my opinion I believe the [documentation on section-crossrefs](https://quarto.org/docs/authoring/cross-references.html#sections) could use some touch-ups on this, particularly to include 

```

## First Section {#sec-first}

This is a playground for Quarto.

{{< lipsum 1 >}}

## Second Section

See [text @sec-first]

or

See [Text](#sec-first)

See @sec-first
```


being actual options:

![grafik](https://github.com/user-attachments/assets/4c0260be-78a6-4e7f-ae94-a22e08dcf1e8)


Currently, only `@sec-first` is explicitly documented. One might infer from the surrounding documentation that `[text](#sec-first)` or `[text @sec-first]` exist, but I regularly find myself looking for what exactly the format is, particularly for `[text](#sec-first)`, when I invariably forget this again. Which then makes me go to old projects after not finding this in the docs themselves.



---
---

P.S.:  
Off-topic, but for GH I am trying to recreate your markdown-table containing the input-doc as a code-cell, and the output as a screenshot. However, I cannot figure out how to wrap the qmd-codeblock without destroying the table itself. Could you give me a screenshot of the raw markdown to create this?

**Reply by mcanouil** [reply]
PR to https://github.com/quarto-dev/quarto-web and documentation issue feedback (https://github.com/quarto-dev/quarto-cli/issues/new/choose).


`````html
<table>
<tr><th>Input</th><th>Output</th></tr>
<tr><td>

````qmd
CODE
````

</td><td>

<!-- IMAGE -->

</td></tr>
</table>


`````

**Comment by Santonia27** [comment]
Thank you thats it!
I overlook this in the documentation.
